### PR TITLE
New menu item hover styles

### PR DESCRIPTION
**Pull request by chris249**
I've added three different menu item hover styles. The first will change nothing (default/as is), second will underline the active/hover menu item in the same color as the text, and the third adds a background to the item on hover/active.

### CODE REVIEW

**Comment by Nikschavan** [comment]
Hey @chris249,

Thank you very much for this PR.

Here is my initial feedback - 

- can you rebase this PR with the `dev` branch and send PR for `dev`?
- CSS should be written using SCSS and all the relevant CSS is generated by the grunt command `grunt minify` you can add the SCSS in [this file](https://github.com/brainstormforce/astra/blob/88f37c069ccf47ab98c5686a1b7a4149cad9f659/sass/site/header/_header-main.scss).

**Comment by chris249** [comment]
Hi @Nikschavan 
I'm not very familiar with GitHub, but do you mean the css will be automatically generated when I submit the SCSS file?

**Comment by chris249** [comment]
@Nikschavan I've edited the scss files and I hope I did it the way it has to be. Can you check it please?

File: inc/extras.php [hunk 1]
```diff
@@ -1238,16 +1239,16 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
 /**
  * Astra Color Palletes.
  */
-if ( ! function_exists( 'astra_color_palette' ) ) :
+if ( ! function_exists( 'astra_color_pallets' ) ) :
```
Comment (Nikschavan) [inline inc/extras.php]: Is this unintended change? This PR should not affect other functions. Right now it creates s fatal error when trying this branch - https://cl.ly/nJkQ

Comment (chris249) [reply inc/extras.php]: @nikschavan, it’s not a change I’ve made, maybe it was there at the time I created the request? I also had some troubles uploading all the files. If you wish I can send you a list with all the files I’ve changed.

**Comment by Nikschavan** [comment]
Closing as this will be implemented in v1.5.0 of Astra Pro

**Pull request closed without merging**